src/precice/impl/SolverInterfaceImpl.cpp

```diff
@@ -757,11 +764,12 @@ int SolverInterfaceImpl:: setMeshEdge
   int secondVertexID )
 {
   TRACE(meshID, firstVertexID, secondVertexID );
+  PRECICE_VALIDATE_MESH_ID(meshID);
   if ( _clientMode ){
     return _requestManager->requestSetMeshEdge ( meshID, firstVertexID, secondVertexID );
   }
   else {
-    CHECK(not _couplingScheme->isInitialized(), "Edges can only be defined before initialize() is called");
```

Comment:
These lines have actually nothing to do with this PR, but with #228 
Will result in a merge conflict.